man/corosync.conf.5

```diff
@@ -809,6 +809,16 @@ enabled when all RT processes are in the root cgroup. So when move_to_root_cgrou
 is disabled and systemd is used, it may be impossible to make systemd options
 like CPUQuota working correctly until corosync is stopped.
 
+Also when this option is used together with cgroup2 and systemd
+it makes impossible (most of the time) for journald to add systemd specific
+metadata (most importantly _SYSTEMD_UNIT) properly, because corosync is
+moved out of cgroup created by systemd. This means
+it is not possible to filter corosync logged messages based on these metadata
+(for example using -u or _SYSTEMD_UNIT=UNIT pattern) and also running
+systemctl status doesn't display (all) corosync log messages.
+Problem is even worse because journald caches pid for some time
```

Comment:
Tiny change (English loves 'the')

"The problem is even worse because journald caches the pid..."